src/Tizen.Multimedia.Util/ThumbnailExtractor/ThumbnailExtractor.cs

```diff
@@ -246,7 +246,10 @@ private static Native.ThumbnailExtractCallback GetCallback(TaskCompletionSource<
         /// </summary>
         /// <remarks>
         /// The size of generated thumbnail will be 320x240.<br/>
-        /// If you want to set the size of generated thumbnail, please use <see cref="Extract(string, Size)"/><br/>
+        /// But, if the size of <paramref name="path"/> has different ratio with 320x240 (approximately 1.33:1),<br/>
+        /// thumbnail will be generated in a way to keep the ratio of <paramref name="path"/>, which based on short axis of <paramref name="path"/>.<br/>
```

Comment:
Suggestion:
       /// thumbnail is generated in a way to keep the ratio of <paramref name="path"/>, which is based on short ...

Reply:
I'll fix it.